source/2_platform/rex_windows/include/rex_windows/platform_creation_params.h

```diff
@@ -14,4 +15,34 @@ namespace rex
     rex::win32::LPtStr cmd_line;
     s32 show_cmd;
   };
+
+  struct GuiParams
+  {
+    s32 window_width      = 1280;
+    s32 window_height     = 720;
+    StringID window_title = "Application"_sid;
+
+    s32 max_render_commands = 1 << 16;
+    s32 max_fps             = 60;
```

Comment:
Is there an issue for these that we should move them into a Config/engine_settings.ini?

Reply:
There's an issue for engine config